rs/bitcoin/adapter/test_utils/src/rpc_client.rs

```diff
@@ -275,7 +275,15 @@ impl<T: RpcClientType> RpcClient<T> {
                     .create_wallet("default", None, None, None, None)
                     .is_err()
                 {
-                    self.load_wallet("default")?;
+                    if let Err(RpcError::JsonRpc(jsonrpc::error::Error::Rpc(
+                        jsonrpc::error::RpcError { code, message, .. },
+                    ))) = self.load_wallet("default")
+                    {
+                        // Wait a second if it says "Wallet already loading."
+                        if code == -4 && message == "Wallet already loading." {
+                            std::thread::sleep(std::time::Duration::from_secs(1));
+                        }
+                    }
```

Comment:
This silently ignores all other errors than `Wallet already loading` errors. Is that intended?

Reply:
You are right! fixed